### ISSUE TITLE
1.19.1 Incompatible?

### ISSUE THREAD

**Issue by piebaker** [astei/krypton#86]
Not sure if this is a 1.19.1 compatible mod, as it only lists 1.19. If it is, then I am encountering an error on launch:

```
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'fabric-networking-api-v1'!
Exit Code: -1
```
[crash-2022-07-31_17.29.44-client.txt](https://github.com/astei/krypton/files/9228115/crash-2022-07-31_17.29.44-client.txt)

**Comment by astei** [comment]
You're running an outdated version of Krypton, 0.2.1 was already released to update for 1.19.1

**Comment by LoganDark** [comment]
Ugh I hate that modrinth still does not display 0.2.1 on the sidebar

![image](https://user-images.githubusercontent.com/4723091/182166364-9f5d291a-b99b-4c17-8804-862e6fcf8a09.png)
